tests/testthat/test_await.R

```diff
@@ -4,14 +4,14 @@ test_that("await captures success_state from status key", {
   f <- function(x) {
     if (x == 0) list(state = "success", y = 2) else list(state = "fail", y = NULL)
   }
-  expect_silent(r <- await(f, x = 0, .status_key = "state",
+  expect_silent(r <- await(f, .x = 0, .status_key = "state",
```

Comment:
It's a good rule of thumb that if we make breaking changes to our own package, the user will experience them too. Because this software gets used a lot, we have a strict 'no breaking changes' philosophy.